### DISCUSSION TITLE
Animation not working on Next.js 13

### DISCUSSION THREAD

**Discussion by iamgideonidoko** [tailwindlabs/tailwindcss#11649] [Help]
**What version of Tailwind CSS are you using?**

v3.3.3

**What build tool (or framework if it abstracts the build tool) are you using?**

Next.js 13.4.10

**What version of Node.js are you using?**

v18.13.0

**What browser are you using?**

Chrome

**What operating system are you using?**

macOS

**Reproduction URL**

https://github.com/vercel/next.js/tree/canary/packages/create-next-app

**Describe your issue**

TailwindCSS animation (like animation-spin) doesn't work in a fresh Next.js 13.4.10 (app directory) project.

**Comment by Oxicode** [comment]
me too

**Reply by gamesmentality** [reply]
I have the same problem. Everything works well but the animations.
